packages/compiler/test/render3/view/binding_spec.ts

```diff
@@ -38,27 +38,31 @@ function makeSelectorMatcher(): SelectorMatcher<DirectiveMeta> {
     inputs: new IdentityInputMapping(['ngForOf']),
     outputs: new IdentityInputMapping([]),
     isComponent: false,
+    selector: '[ngFor][ngForOf]'
```

Comment:
nit: could you add trailing commas to the lines you added in this file

```suggestion
    selector: '[ngFor][ngForOf]',
```